server/mscalendar/daily_summary_test.go

```diff
@@ -257,9 +257,9 @@ Wednesday February 12, 2020
 					mockPoster.EXPECT().DM("user2_mm_id", `Times are shown in Pacific Standard Time
 Wednesday February 12, 2020
 
-| Time | Subject |
-| :--: | :-- |
-| 9:00AM - 11:00AM | [The subject]() |`).Return("postID2", nil).Times(1),
+| Time | Subject | |
+| :-- | :-- | :-- |
+| 9:00AM - 11:00AM | [The subject]() |  |`).Return("postID2", nil).Times(1),
```

Comment:
LGTM, though maybe worth adding the meeting link/location here for testing the new functionality?